src/be/app.py

```diff
@@ -37,7 +41,19 @@ def status():
 
 @app.post("/api/v1/turn_on")
 def turn_on():
+    
     # TODO turn bulb on here
     return "Hello from the server! implement this function"
 
+bulb = None
+
+def discover_bulb():
+    global bulb
+    bulbs = discover_bulbs()
+    bulb_ip = bulbs[0]['ip']
+    bulb = bulb_ip
+
+discover_bulb()
+
+
```

Comment:
I imagine your next step would be to add a function like:

```python
def get_status(bulb: yl.Bulb) -> Status:
    bulb.get_properties() # or w/e
    # get values for colour, brightness, power_status
    power_status = ...
    colour = ...
    brightness = ....
    return Status(brightness=brightness, colour=colour, power_status=power_status)
```

then in the get_status endpoint (the function decorated with `app.get("/api/v1/status`) you can just use your function implementation above to return the status.

Reply:
Might be a good idea haha